kvdb-rocksdb/src/lib.rs

```diff
@@ -235,6 +237,86 @@ impl DBAndColumns {
 	}
 }
 
+struct DbStats {
+	reads: AtomicU64,
+	writes: AtomicU64,
+	write_bytes: AtomicU64,
+	read_bytes: AtomicU64,
+	transactions: AtomicU64,
+	started: Mutex<std::time::Instant>,
```

Comment:
Could be an `RwLock` for better performance

Reply:
Sure, I'll update, but It is invoked only for getting/resetting accumulated statistics, which supposed to be  quite rare